setup.py

```diff
@@ -150,13 +162,22 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
         file.write(text)
 
 
-if os.environ.get('USE_THEME_FROM_GIT'):
-    _SPHINX_AIRFLOW_THEME_URL = (
-        "@ https://github.com/apache/airflow-site/releases/download/0.0.4/"
-        "sphinx_airflow_theme-0.0.4-py3-none-any.whl"
-    )
-else:
-    _SPHINX_AIRFLOW_THEME_URL = ''
+def get_sphinx_theme_version() -> str:
+    """
+    Return sphinx theme version. If USE_THEME_FROM_GIT env variable is set, the theme is used from
+    github tp allow dynamically update it during development. However for regular PIP release
+    you cannot use @ package specification, so the latest available released theme package from
+    PIP is used.
+    :return: the
+    """
+    if os.environ.get('USE_THEME_FROM_GIT'):
+        return (
+            "@ https://github.com/apache/airflow-site/releases/download/0.0.4/"
+            + "sphinx_airflow_theme-0.0.4-py3-none-any.whl"
+        )
+    else:
+        return ''
```

Comment:
```suggestion
    return ''
```

`else` is not required here as if the first condition is True, it will return

Reply:
True.

Reply:
Also removed same with git_version above.